src/CredentialsWrapper.php

```diff
@@ -54,7 +54,7 @@ class CredentialsWrapper
     use ValidationTrait;
 
     /** @var FetchAuthTokenInterface $credentialsFetcher */
-    private FetchAuthTokenInterface $credentialsFetcher;
+    private ?FetchAuthTokenInterface $credentialsFetcher = null;
```

Comment:
This may've been added by mistake?

Reply:
No, it was required in order for the tests to pass - this was throwing an error when running these tests (see [this failing test](https://github.com/googleapis/gax-php/actions/runs/6789914294/job/18458150731#step:6:1569))